.github/workflows/release.yml

```diff
@@ -17,7 +17,6 @@ jobs:
         with:
           node-version: '18'
           registry-url: 'https://registry.npmjs.org'
-          cache: 'npm'
```

Comment:
Why are we killing the cache? Are all caches a security risk?

Reply:
Just for releases, when workflows publish artifacts - https://woodruffw.github.io/zizmor/audits/#cache-poisoning

Reply:
Thanks for explaining